packages/blocks/src/page-block/edgeless/components/block-portal/note/edgeless-note.ts

```diff
@@ -62,29 +63,72 @@ export class EdgelessNoteMask extends WithDisposable(ShadowlessElement) {
 
 @customElement('edgeless-block-portal-note')
 export class EdgelessBlockPortalNote extends EdgelessPortalBase<NoteBlockModel> {
+  @state()
+  private _editing = false;
+
+  @state()
+  private _transition = 'none';
+
+  override firstUpdated() {
+    const selection = this.surface.edgeless.selectionManager;
+    selection.slots.updated.on(async () => {
```

Comment:
Here it's suggested to pipe it into `this.disposables` so that it could be disposed